dist/src/main/resources/configuration/hawkular-realm-for-dev.json

```diff
@@ -86,6 +86,8 @@
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": false,
+      "redirectUris": ["/*", "http://localhost:2772/*"],
```

Comment:
It's relative to the main URL, so, it will allow the same protocol, hostname and port as the auth server (ie: Keycloak).